### ISSUE TITLE
fixLayoutHeight

### ISSUE THREAD

**Issue by pucoder** [ColorlibHQ/AdminLTE#3541]
**Environment (please complete the following information):**
- AdminLTE Version: [e.g. v3.1.0]

I used vue2.6.12 to introduce admin-lte3.1.0. After the page jump, there would be extra white space at the bottom. I tried to call the method, but it didn't work,What should I do, please? 

```vue
    mounted() {
        $(document).Layout('fixLayoutHeight');
    },
```

**Comment by danny007in** [comment]
Give some more details regarding to issue like code, images, video to solve issue

**Comment by REJack** [comment]
fixed with 4a2a28f
